Jellyfin.Server/Migrations/Routines/MigrateActivityLogDb.cs

```diff
@@ -76,17 +77,38 @@ public void Perform()
                 dbContext.Database.ExecuteSqlRaw("UPDATE sqlite_sequence SET seq = 0 WHERE name = 'ActivityLog';");
                 dbContext.SaveChanges();
 
-                var newEntries = queryResult.Select(entry =>
+                var newEntries = new List<ActivityLog>();
+
+                foreach (var entry in queryResult)
                 {
                     if (!logLevelDictionary.TryGetValue(entry[8].ToString(), out var severity))
                     {
                         severity = LogLevel.Trace;
                     }
 
+                    Guid guid;
+                    if (entry[6].SQLiteType == SQLiteType.Null)
+                    {
+                        guid = Guid.Empty;
+                    }
+                    else if (!Guid.TryParse(entry[6].ToString(), out guid))
+                    {
+                        // This is not a valid Guid, see if it is an internal ID from an old Emby schema
+                        var userEntry = userDbConnection
+                            .Query($"SELECT guid FROM LocalUsersv2 WHERE Id = {entry[6].ToString()}")
```

Comment:
shouldn't we be using data binding here instead of shoving raw data into the select?